var/spack/repos/builtin/packages/w3emc/package.py

```diff
@@ -14,20 +14,39 @@ class W3emc(CMakePackage):
 
     homepage = "https://noaa-emc.github.io/NCEPLIBS-w3emc/"
     url = "https://github.com/NOAA-EMC/NCEPLIBS-w3emc/archive/refs/tags/v2.9.0.tar.gz"
+    git = "https://github.com/NOAA-EMC/NCEPLIBS-w3emc"
 
     maintainers("t-brown", "AlexanderRichert-NOAA", "Hang-Lei-NOAA", "edwardhartnett")
 
+    version("develop", branch="develop")
     version("2.10.0", sha256="366b55a0425fc3e729ecb9f3b236250349399fe4c8e19f325500463043fd2f18")
     version("2.9.3", sha256="9ca1b08dd13dfbad4a955257ae0cf38d2e300ccd8d983606212bc982370a29bc")
     version("2.9.2", sha256="eace811a1365f69b85fdf2bcd93a9d963ba72de5a7111e6fa7c0e6578b69bfbc")
     version("2.9.1", sha256="d3e705615bdd0b76a40751337d943d5a1ea415636f4e5368aed058f074b85df4")
     version("2.9.0", sha256="994f59635ab91e34e96cab5fbaf8de54389d09461c7bac33b3104a1187e6c98a")
     version("2.7.3", sha256="eace811a1365f69b85fdf2bcd93a9d963ba72de5a7111e6fa7c0e6578b69bfbc")
 
+    variant("pic", default=True, description="Build with position-independent-code")
     variant("bufr", default=False, description="Build with BUFR routines", when="@2.10:")
+    variant(
+        "precision",
+        default=["4", "d"],
+        values=["4", "d", "8"],
+        multi=True,
+        description="Set precision (_4/_d/_8 library versions)",
+        when="@2.10:",
+    )
+    variant("shared", default=False, description="Build shared library", when="@2.10:")
+    variant(
+        "extradeps",
+        default=False,
+        description="Build w3emc with subprograms which call unknown dependencies",
```

Comment:
```suggestion
        description="Build w3emc with subprograms which call unknown dependencies",
```
What do you mean by "unknown dependencies"? :slightly_smiling_face:

Reply:
lol good question... There's some old code in there that calls routines that no one knows where they come from, so we want that off by default, but leave the option for development purposes.